Gemfile

```diff
@@ -54,7 +54,12 @@ group :test do
   gem 'selenium-webdriver'
   # Easy installation and use of web drivers to run system tests with browsers
   gem 'webdrivers'
+  #Add SimpleCov
+  gem 'simplecov', require: false
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+#Add Rubocop
+gem 'rubocop', '~> 1.25', require: false
```

Comment:
You added the Rubocop gem to the project but haven't set it up and and there are still a lot of offenses in it. Also, as this is a Rails project using RSpec, you should also include the `rubocop-rspec` and `rubocop-rails` gems to include cops specific for those libraries.